api/config/gateway/v1/gateway.proto

```diff
@@ -85,6 +85,23 @@ message GatewayOptions {
   Timeouts timeouts = 5;
 
   repeated Middleware middleware = 6;
+
+  Assets assets = 7;
+}
+
+// Assets configuration provide a passthrough host for frontend static assets.
+// This is useful if you dont have the ability to enable sticky sessions or a blue/green deploy system in place.
+message Assets {
+  // To use the S3Provider you must have the AWS service configured
+  message S3Provider {
+    string region = 1;
+    string bucket = 2;
+    string key = 3;
```

Comment:
what is key in this context? is it actually prefix? add comment.

Reply:
This is just S3 terminology, I could rename to `path` if that makes more sense although that does not alight with S3 terms. Otherwise ill just comment.

Reply:
i think of key as the entire path following bucket, whereas afaik this represents everything before the requested filename, and then the filename is appended to it. comment is enough.